drivers/hmis/app/graphql/types/hmis_schema/ce_referral_event.rb

```diff
@@ -0,0 +1,38 @@
+###
+# Copyright 2016 - 2025 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+# frozen_string_literal: true
+
+module Types
+  class HmisSchema::CeReferralEvent < Types::BaseObject
+    # object is a Hmis::WorkflowExecution::AuditEvent
+    field :id, ID, null: false
+    field :created_at, GraphQL::Types::ISO8601DateTime, null: false
+    field :user, Types::Application::User, null: true
+    field :type, String, null: false, method: :event_type
+    field :step_name, String, null: true
+
+    def step_name
+      step = load_ar_association(object, :step)
+      return nil unless step.present?
+
+      load_ar_association(step, :node).name
+    end
+
+    def type
+      case object.event_type
+      when 'complete_step'
+        'Completed Task'
+      when 'start_workflow'
+        'Started Referral'
+      when 'end_workflow'
+        object.event_data['result'] == 'reject_referral' ? 'Declined Referral' : 'Accepted Referral'
+      else
+        ''
+      end
```

Comment:
I would suggest resolving these as an Enum, so the frontend can differentiate them eventually with type safety (Rather than checking against arbitrary strings). For example it might want to show Declined in red, Accepted in green, etc.